scripts/make_empty_repo.sh

```diff
@@ -51,5 +50,4 @@ CURATED_CALIB="${STD_CALIB}/curated"
 
 butler write-curated-calibrations "$REPO_DIR" $INST_CLASS \
     --collection "$CURATED_CALIB"
-butler collection-chain "$REPO_DIR" "$STD_CALIB" "$CURATED_CALIB"
```

Comment:
What is the purpose of removing this? You don't seem to add a corresponding line anywhere else.

Reply:
I am suprised you see this, because I forced pushed 2 hours ago, and this file is not longer modified in the PR.

Reply:
In the future, please make such changes in either a fixup commit or by rebase-editing the original commit, instead of bundling them with unrelated changes; doing so will make the history much easier to follow.

See the [developer guide](https://developer.lsst.io/work/flow.html#git-commit-organization-best-practices) for guidelines about organizing commits (the OpenStack guide is so-so; it talks about what to do with aggregated commits but not what to do if they're shredded to the point of not being self-contained).